CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+1.13.0 (in progress)
+====================
+
+New Features
+------------
+
+- Previously, if a Room was completed via the REST API while your
```

Comment:
@markandrus 
```
- When a Room is completed using the REST API, the Room emits a "disconnected" event with a new parameter - a TwilioError with code 53118 and message "Room completed". In some applications, this will be expected, so you should set an event listener on the Room as follows:
```
?